### DISCUSSION TITLE
Configuration to hide Wins-Draws-Losses

### DISCUSSION THREAD

**Discussion by narfotic** [vasqued2/ha-teamtracker-card#64] [Ideas]
Would it be possible to add an option in the configuration to hide Wins-Draws-Losses under the team names?

**Comment by vasqued2** [comment]
You should be able to use [card-mod](https://github.com/thomasloven/lovelace-card-mod) to hide it.  [This Wiki entry](https://github.com/vasqued2/ha-teamtracker/wiki/Using-CARD-MOD-card-to-format-the-TEAMTRACKER-CARD) has examples that show how to hide specific elements on the card.  Just use it to hide the `record` element and you should be able to do what you want without any changes on my side.  I decided this gave a lot more options to change the look than continually adding additional switches and options to the card configuration.

Lmk if you have problems getting it to work.

[https://github.com/vasqued2/ha-teamtracker/wiki/Using-CARD-MOD-card-to-format-the-TEAMTRACKER-CARD](https://github.com/vasqued2/ha-teamtracker/wiki/Using-CARD-MOD-card-to-format-the-TEAMTRACKER-CARD)
